### ISSUE TITLE
training detail & inference time of the model

### ISSUE THREAD

**Issue by jjldr** [mrnabati/CenterFusion#16]
Hi, I train the model from scratch(use pretrained dla34 in imagenet)，but the training loss is about 12 after 50 epochs, the loss is too huge. In your code, you freeze the dla34 backbone when training, is it the reason why I can not get a lower loss?
another question is the inference time of the model. When I test your model in Tesla V 100 GPU， the inference time of one sample is about 0.37s, it is slower than original centerNet model. centerNet is real-time model, but your model is too slow to be real-time 
hope for your answer, thanks

**Comment by mrnabati** [comment]
Hi. I did not try training from scratch. I started from the pre-trained CenterNet model which is trained for 140 epochs, and trained it for another 60 epochs. So you probably need to train for about 200 epochs to get to the same loss values. I froze the backbone mostly to increase training speed, but it might have some effect on the loss values as well.
As for the inference time, it is slower than the original CenterNet. The bottleneck is mostly the way the [generate_pc_hm()](https://github.com/mrnabati/CenterFusion/blob/616dcda9464c17f98b1550ea8a212bce30eba1aa/src/lib/utils/pointcloud.py#L208) function is implemented. There are more efficient ways to do it, but I just haven't had time to do it yet.
